{"name": "用例9：观众端申请上麦(通过点击交友位)", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\pgk bqt\\\\PycharmProjects\\\\Chamet_project\\\\common/screenshots/party_ten/party_ten_2023-06-26 11_51_11.png'", "trace": "self = <test_case.test_10party_mult.Test_multchamet object at 0x0000017F695F8710>\n\n    @allure.severity(allure.severity_level.BLOCKER)\n    @allure.title('用例9：观众端申请上麦(通过点击交友位)')\n    def test_TenParty_009(self):\n        logging.info('===用例9：观众端申请上麦(通过点击交友位)===')\n        try:\n            if self.multaudience.partysite_applyguest(1):\n                assert True\n                logging.info('===断言成功，申请上麦成功===')\n            else:\n>               raise\nE               RuntimeError: No active exception to reraise\n\ntest_10party_mult.py:197: RuntimeError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_case.test_10party_mult.Test_multchamet object at 0x0000017F695F8710>\n\n    @allure.severity(allure.severity_level.BLOCKER)\n    @allure.title('用例9：观众端申请上麦(通过点击交友位)')\n    def test_TenParty_009(self):\n        logging.info('===用例9：观众端申请上麦(通过点击交友位)===')\n        try:\n            if self.multaudience.partysite_applyguest(1):\n                assert True\n                logging.info('===断言成功，申请上麦成功===')\n            else:\n                raise\n        except AssertionError as e:\n            logging.info('===断言失败===')\n            screen_name = self.multaudience.screenshot('party_ten')\n            logging.info(f'截图成功，图片为{screen_name}')\n            raise\n        except:\n            logging.info('===执行失败===')\n>           screen_name = self.multaudience.screenshot('party_ten')\n\ntest_10party_mult.py:205: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <common.partytencommon.multparty_ten_audience.Multaudience object at 0x0000017F69612B90>, module = 'party_ten'\n\n    def screenshot(self, module):\n        time = self.getTime()\n        image_file = os.path.dirname(os.path.dirname(__file__)) + '/screenshots/party_ten/%s_%s.png' % (module, time)\n        self.driver.get_screenshot_as_file(image_file)\n>       with open(image_file, mode='rb') as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\pgk bqt\\\\PycharmProjects\\\\Chamet_project\\\\common/screenshots/party_ten/party_ten_2023-06-26 11_51_11.png'\n\n..\\common\\partytencommon\\multparty_ten_audience.py:76: FileNotFoundError"}, "attachments": [{"name": "log", "source": "7537976d-7857-4e7f-80ac-4195adc80926-attachment.txt", "type": "text/plain"}], "start": 1687751461261, "stop": 1687751472793, "uuid": "7b15febd-2bf1-4bfe-987e-4afaf7ac6df0", "historyId": "9a9fad1367f1f75b0bb19f879f4fb845", "testCaseId": "9a9fad1367f1f75b0bb19f879f4fb845", "fullName": "test_10party_mult.Test_multchamet#test_TenParty_009", "labels": [{"name": "severity", "value": "blocker"}, {"name": "suite", "value": "test_10party_mult"}, {"name": "subSuite", "value": "Test_multchamet"}, {"name": "host", "value": "DESKTOP-8OL56K6"}, {"name": "thread", "value": "14112-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_10party_mult"}]}